### ISSUE TITLE
Corporate Sponsors - remove extraneous [JG1] text

### ISSUE THREAD

**Issue by terrimorton** [dotnet-foundation/website#298]
Page: https://dotnetfoundation.org/member/corporate-sponsors

I found this issue when running Accessibility Insights for Web on this page. The extraneous "[JG1]" text should be removed from the end of the "Why change the TSG to a Corporate Sponsor Program?" paragraph.

Marked up screenshot:
![image](https://user-images.githubusercontent.com/4643903/89971608-c4198c80-dc29-11ea-9983-d21e083aa69c.png)

**Comment by Arhell** [comment]
@terrimorton done
